app/views/sessions/show_cart.html.erb

```diff
@@ -1,12 +1,52 @@
-Cart:&nbsp;<%= session[:cart] %> <br>
-<% if session[:cart] %>
-<% session[:cart].each do |item_id, quantity|%>
-    <% item = Item.find_by(id: item_id) %>
-    <% if item %>
-    <%= item.name %>
-    <%= item.price %>
-    <%= quantity %>
-    <%= button_to "Remove from cart", delete_from_cart_path(item) %>
-<% end %>
-<% end %>
-<% end %>
+<h1>Cart</h1>
+ <table WIDTH="50%" BORDER="2" ALIGN="CENTER" CELLSPACING="40">
+        <tr ALIGN="Center">
+            <td>Item Seller</td>
+            <td>Item Name</td>
+            <td>Price</td>
+            <td>Quantity</td>
+        </tr>
+        <% if session[:cart] %>
+            <% session[:cart].each do |item_id, quantity| %>
+            <% item = Item.find_by(id: item_id) %>
+                <tr>
+                    <td>
+                        <%= item.user.first_name %> <%= item.user.last_name %>
+                    </td>
+                    <td>
+                        <%= item.name %>
+                    </td>
+                    <td>
+                        <%= item.price %>
+                    </td>
+                    <td>
+                        <%= quantity %>
+                    </td>
+                </tr>
+            <% end %>
+        <% end %>
+        <tr>
+            <td>
+                Order Total:
+            </td>
+            <td>
+            </td>
+            <% if session[:cart] %>
+                <% total_price = 0 %>
+                <% total_quantity = 0 %>
+                <% session[:cart].each do |item_id, quantity| %>
+                    <% item = Item.find(item_id) %>
+                    <% total_quantity += quantity %>
+                    <% total_price += (item.price * quantity) %>
+                <% end %>
+                <td>
+                    <%=total_price%> 
+                </td>
+                <td>
+                   <%=total_quantity%>
+                </td>
+            <% end %>
+        </tr>
+    </table>
+    <br>
+    <%= link_to "Checkout", checkout_path, class: "btn pull-right btn-primary" %>
```

Comment:
@BenDrews Especially with HTML, since the innate structure of the language is really,really hard to read easily. Be very careful with things like indentation. If at all possible. Aim for symmetry.The fact that this ends, suddenly, on a tabbed line, is problematic.

_Though I want to emphasize. Looking at this code. It is generally very well organized._